components/Flight.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import { FlatList, Text, View } from 'react-native';
+import React, { useEffect } from 'react';
+import { FlatList, ImageBackground, Text, View } from 'react-native';
```

Comment:
I removed the background as part of the re-styling. Remind me to clean up unused imports/variables later please.

Reply:
kept the change for the useEffect, didn't notice the ImageBackground. can be removed

Reply:
removed